bindings/python/src/lib.rs

```diff
@@ -145,7 +145,7 @@ impl Cursor {
     }
 
     pub fn fetchall(&mut self, py: Python) -> Result<Vec<PyObject>> {
-        let mut results = Vec::new();
+        let mut results = vec![];
```

Comment:
Why? what is wrong with using the normal function?

Reply:
I think `vec![]` is more idiomatic Rust, that's all.